lib/r10k/source/git.rb

```diff
@@ -111,9 +117,25 @@ def desired_contents
 
   private
 
+  def filter_branches(branches, ignore_prefixes)
+    filter = Regexp.new("^(#{ignore_prefixes.join('|')}).?")
```

Comment:
What is the `.?` doing in the regex? I assume it is an optional `any character`, but I'm not sure why it is needed.

Reply:
It is not. Removing.